custom/src/main/java/org/kiwix/kiwixmobile/custom/main/CustomReaderFragment.kt

```diff
@@ -107,9 +107,10 @@ class CustomReaderFragment : CoreReaderFragment() {
     val urls = JSONArray(zimArticles)
     val zimPositions = JSONArray(settings.getString(TAG_CURRENT_POSITIONS, null))
     selectTab(currentTab)
-    loadUrlWithCurrentWebview(UpdateUtils.reformatProviderUrl(urls.getString(currentTab)))
-    getCurrentWebView().scrollY = zimPositions.getInt(currentTab)
-  }
```

Comment:
Accidental removal of bracket.

e: /Users/runner/work/kiwix-android/kiwix-android/custom/src/main/java/org/kiwix/kiwixmobile/custom/main/CustomReaderFragment.kt: (236, 2): Missing '}'

Reply:
Considered! thanks for help :-)